pkgs/ffigen/example/ffinative/lib/generated_bindings.dart

```diff
@@ -1,22 +1,26 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
 // ignore_for_file: deprecated_member_use
 
 // AUTO GENERATED FILE, DO NOT EDIT.
 //
 // Generated by `package:ffigen`.
 // ignore_for_file: type=lint
 import 'dart:ffi' as ffi;
+import '' as self;
 
 /// Adds 2 integers.
 @ffi.Native<ffi.Int Function(ffi.Int, ffi.Int)>(
-    symbol: 'sum', assetId: 'package:ffinative_example/generated_bindings.dart')
+    assetId: 'package:ffinative_example/generated_bindings.dart')
```

Comment:
Nice, reusing the symbol.

Side note which does not have to be addressed in this CL: Should we consider generating

```dart
@DefaultAsset('package:ffinative_example/generated_bindings.dart')
library;
```

instead of an `assetId` in every `@Native`?

Reply:
I think we should do that, especially considering that there can only be a single `assetId` in the config file at the moment. But even if we want different asset ids based on symbol patterns, we can always specify the id for those symbols only.

Reply:
Please file an issue for us to do this later, or add it in this PR.

Reply:
I've removed the `assetId` to generate a `@DefaultAsset` annotation instead.